assets/src/js/plugins/give-api/api.js

```diff
@@ -106,7 +106,7 @@ const Give = {
 			} else {
 				// Properly position symbol after if selected.
 				if ( 'after' === args.position ) {
-					args.format = '%v%s';
+					args.format = '%v %s';
```

Comment:
The documentation for the [accounting.js#formatMoney](http://openexchangerates.github.io/accounting.js/#methods) library uses the example format of `"%v %s"`, so this looks good to me.

@ravinderk it looks like you initially wrote this, but it was 2 years ago. Do you recall any specific reason to not include the space here, or is this an old typo?

Reply:
@kjohnson I agree with @karlsauter. All currencies do not have the same formatting: https://github.com/impress-org/givewp/issues/5367#issuecomment-736840586. We can add a property to currency config which will tell us whether or not to add space between currency icon/code and amount.

This formatting will break for another currency formatting like `USD`.

Reply:
Ok, so it sounds like we will need to take a different approach.

Thank you, @ravinderk for taking a look.